engines/griffon/configure.engine

```diff
@@ -0,0 +1,3 @@
+# This file is included from the main "configure" script
+# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps]
+add_engine griffon "The Griffon Legend" no
```

Comment:
The configuration is missing the setting.. "highres" I think? to indicate that an engine uses a high color mode

Reply:
Good catch. Added "16bit" there